packages/mui-codemod/src/v5.0.0/rename-css-variables.test.js

```diff
@@ -0,0 +1,29 @@
+import path from 'path';
+import { expect } from 'chai';
+import jscodeshift from 'jscodeshift';
+import transform from './rename-css-variables';
+import readFile from '../util/readFile';
+
+function read(fileName) {
+  return readFile(path.join(__dirname, fileName));
+}
+
+describe('@mui/codemod', () => {
+  describe('v5.0.0', () => {
+    describe('rename-css-variables', () => {
+      it('renames css variables based on the new rules', () => {
+        const actual = transform(
+          {
+            source: read('./rename-css-variables.test/actual.js'),
+            path: require.resolve('./joy-rename-components-to-slots.test/actual.js'),
+          },
+          { jscodeshift },
+          {},
+        );
+
+        const expected = read('./rename-css-variables.test/expected.js');
+        expect(actual).to.equal(expected, 'The transformed version should be correct');
+      });
```

Comment:
Can you add another test to make sure that the codemod is idempotent?

Here is an example https://github.com/mui/material-ui/blob/master/packages/mui-codemod/src/v5.0.0/box-rename-css.test.js#L28.

Reply:
Good to know! Done! The codemod passes the test.